notebooks/growth_model_classification.ipynb

```diff
@@ -0,0 +1,1025 @@
+{
```

Comment:
<p>I think we should add a line to describe what this plot shows:</p><p><code>In the above plot, we try to visualize the 3 different cluster groups by plotting their slope values (along x-axis) and their median predicted value (along y-axis).</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>